docker-compose.yml

```diff
@@ -37,9 +37,9 @@ services:
       dockerfile: Dockerfile
     restart: always
     volumes:
-      - ./airflow_config/dags:/usr/local/airflow/dags
-      - ./airflow_config/requirements.txt:/requirements.txt
-      - ./airflow_config/plugins:/usr/local/airflow/plugins
+      - ./cornflow-dags/DAG:/usr/local/airflow/dags
+      - ./cornflow-dags/requirements.txt:/requirements.txt
+      - ./cornflow-server/airflow_config/plugins:/usr/local/airflow/plugins
```

Comment:
This last line, should not need to be changed as the context of the docker compose file is `./cornflow-server/airflow_config`, no?

Reply:
This same thing could be said about the rest of the docker compose files

Reply:
Now, the context of docker-compose files is in root repo folder. We have to load plugins from ./cornflow-server/airflow_config/plugins.
For DAG and requirements, now been moved to new DAG folder in the repo we load them from ./cornflow-dags/DAG and ./cornflow-dags/requirements.txt.